gitlab_jobs.py

```diff
@@ -100,7 +113,7 @@ def main():
         duration_min = pipeline.duration / 60.0
         print(template.format(
             duration_min=duration_min, **pipeline.attributes))
-        for job in pipeline.jobs.list(scope='success', all=True):
+        for job in get_jobs(pipeline, scope='success', all=True):
```

Comment:
Hey, wait a minute.  I thought `all=True` meant I would be getting all the pages without manually iterating over them?
https://python-gitlab.readthedocs.io/en/stable/api-usage.html#pagination

I don't have any GitLab projects with sufficient numbers of build jobs to require pagination.  Have you such a project, and did the old code fail for it?

Reply:
My experiments indicate that yes, all returns everything, so this PR actually introduced a bug that causes repeated jobs :(

Fixed in 7b4965cf1ef0e3e324096f52b251e70ddd719356.